build/_k-test.shade

```diff
@@ -3,6 +3,7 @@ use import="Environment"
 
 default NO_PARALLEL_TEST_PROJECTS='${E("NO_PARALLEL_TEST_PROJECTS")}'
 default KOREBUILD_TEST_DNXCORE='${E("KOREBUILD_TEST_DNXCORE")}'
+default CORECLR_RUNTIME_VERSION='${E("CORECLR_RUNTIME_VERSION")}'
```

Comment:
This is not true. For dnx451 we run just `cmd /C dnx test` while for dnxcore50 we run ` cmd /C "C:\Source\dnx\packages\KoreBuild\build\dnvm" run default -runtime coreclr test`. So we run tests differently. On top of that - because we use dnvm in the second case the build will pass even if tests failed because the original exit code is __lost__.